astropy/stats/info_theory.py

```diff
@@ -379,7 +379,7 @@ def akaike_info_criterion_lsq(ssr, n_params, n_samples):
     >>> ssr_g2 = np.sum((g2_fit(x) - y)**2.0)
     >>> ssr_g1 = np.sum((g1_fit(x) - y)**2.0)
     >>> akaike_info_criterion_lsq(ssr_g3, 9, x.shape[0]) # doctest: +FLOAT_CMP
-    -656.32589850659224
+    -660.41075962620482
```

Comment:
Hey @pllim, sorry, I didn't notice you mentioned me here.
Hmm, it's surprising that only this value changed...
But this shouldn't hold this PR. :)